app/javascript/miq-redux/store.ts

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware, compose, Store} from 'redux';
+import {rootReducer} from './reducer';
+
+import {AppState} from '../packs/miq-redux';
+
+const composeEnhancers = window['.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose;
```

Comment:
you are correct, I didnt see that example, in general its not required if you don't provide an initialState but that make sense long term too (caching in session storage etc).

just a question, why do you access it via [] vs  
```js
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```